crates/matrix-sdk/src/room/mod.rs

```diff
@@ -3236,6 +3371,17 @@ impl Room {
             };
         Ok(current_join_request_ids)
     }
+
+    /// Subscribes to the set of requests to join that have been marked as
+    /// 'seen'.
+    pub async fn subscribe_to_seen_join_request_ids(
```

Comment:
I think this isn't tested in isolation. Maybe we can inline this, and not make it public?